templates/feedback.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'layouts/_base.html' %}
```

Comment:
Last thing, the exit button does not link to anything and tries to submit the form instead.